buildSrc/fetchDictionaries.js

```diff
@@ -62,16 +62,16 @@ async function publishDebPackage() {
 	const electronVersion = await getElectronVersion()
 	const deb = `tutanota-desktop-dicts_${electronVersion}_amd64.deb`
 
-	console.log("create " + debs.dict)
+	console.log("create", deb)
 	exitOnFail(spawnSync("/usr/local/bin/fpm", `${commonArgs} -n tutanota-desktop-dicts -v ${electronVersion} dictionaries/=${target}-desktop/dictionaries`.split(" "), {
-		cwd: __dirname + "/build",
+		cwd: path.join(process.cwd(), "build"),
```

Comment:
__dirname was actually undefined in my local runs and in the node REPL. with this PR the deb gets created in `tutanota-3/build/` (since we're running the build from the repo's root, right?) and then copied to the `/opt/` destination. the first location is not really important, though.

Reply:
`__dirname` doesn't exist in ESM but you can get it
`const __dirname = path.dirname(fileURLToPath(import.meta.url))`

Reply:
cursed

Reply:
I think it was `__dirname` because this code was part of the `dist.js` script before which lived in the repo root. I think this version is better.

Reply:
there's no point in doing `cwd + build`, you can just pass `build` in this case